test/config/sa.json

```diff
@@ -24,7 +24,8 @@
       ]
     },
     "features": {
-      "DeleteUnusedChallenges": true
+      "DeleteUnusedChallenges": true,
+      "WriteIssuedNamesPrecert": true
```

Comment:
Is there a reason to include the feature flag in `config/sa.json` if it's deprecated and doesn't have any affect or can it just be deleted ala `config-next/`?